app/models/vim_performance_analysis.rb

```diff
@@ -159,7 +159,7 @@ def vm_how_many_more_can_fit(options = {})
             details = []
             total   = 0
             storages_for_compute_target(c).each do |s|
-              avail, usage = storage_offers(compute_perf, ts, options[:target_options][:storage], s)
+              avail, usage = compute_offers(compute_perf, ts, options[:target_options][:storage], :storage, s)
```

Comment:
aah

ok, so both will work, but it looks like just calling offers (for both?) would make more sense.